src/extra/normalize.css

```diff
@@ -37,7 +37,11 @@
   outline-offset: 5px;
 }
 
-:where(body) { 
+:where(input[type='text']:not([disabled]):focus) {
+  outline: none;
+}
```

Comment:
sorry, the issue is bigger than just text inputs, it's just what is shown in the video. were you able to reproduce the issue on the text input?